src/components/Wrapper.tsx

```diff
@@ -7,8 +7,11 @@ import ClientsCarousel from "./ClientsCarousel";
 import YogaBanner from "./YogaBanner";
 import PositiveYogaList from "./PositiveYogaList";
 import Questions from "./Questions";
```

Comment:
if you've had the index.ts in your components,you wouldn't need so many imports. you could just simply import {YogaBanner, PositiveYogaList, Questions} from './components';